pom.xml

```diff
@@ -104,6 +104,14 @@
             <scope>system</scope>
             <systemPath>${project.basedir}/lib/LibsDisguises-9.0.9.jar</systemPath>
         </dependency>
+
```

Comment:
Very much less keen then for us to work with the plugin if we cannot find a way of automating the Maven integration, as it'll basically push back weeks of work elsewhere in the plugin where we've tried to automate these things ready for CI...